kraft run --network bridge panics with a bound check error when no interface present

### Describe the bug

I have built a fc target of https://github.com/unikraft/app-python3

I created a bridge (kraft0), but didn't add an interface.

I attempted to run it with:

```console
kraft run .unikraft/build/kraftcloud-x86_64_fc-x86_64 --plat fc --network bridge:kraft0
```

kraft panics with:

`panic: runtime error: index out of range [0] with length 0`

The error message could be more helpful.

### Steps to reproduce

_No response_

### Expected behavior

An interface should be created or the message made more helpful.

### Which architectures were you using or does this bug affect?

_No response_

### Which operating system were you using or does this bug affect?

_No response_

### Relevant log output

```shell
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
kraftkit.sh/machine/network/bridge.(*v1alpha1Network).Get(0x0?, {0x0?, 0x0?}, 0xc0001f6b40)
	/home/jake/unikraft/kraftkit/machine/network/bridge/v1alpha1.go:450 +0x378
api%2ezip.(*MethodClient[...]).Do(0x2608880, {0x2615568, 0xc00045f7d0}, 0xc0001f6b40)
	/home/jake/go/pkg/mod/api.zip@v0.1.5/client.go:49 +0xe8
kraftkit.sh/api/network/v1alpha1.(*NetworkServiceHandler).Get(0xd9bf6b?, {0x2615568?, 0xc00045f7d0?}, 0xc00053b340?)
	/home/jake/unikraft/kraftkit/api/network/v1alpha1/network.zip.go:151 +0x2c
kraftkit.sh/cmd/kraft/run.(*Run).parseNetworks(0xc00052a420, {0x2615568, 0xc00045f7d0}, 0xc0002ad080)
	/home/jake/unikraft/kraftkit/cmd/kraft/run/utils.go:43 +0xcf
kraftkit.sh/cmd/kraft/run.(*Run).Run(0xc00052a420, 0x0?, {0xc000112190, 0x1, 0x5})
	/home/jake/unikraft/kraftkit/cmd/kraft/run/run.go:300 +0xb2e
kraftkit.sh/cmdfactory.bind.func1(0xc000524900?, {0xc000112190, 0x1, 0x5})
	/home/jake/unikraft/kraftkit/cmdfactory/builder.go:689 +0x19d
kraftkit.sh/cmdfactory.bind.func1(0xc000524900?, {0xc000112190, 0x1, 0x5})
	/home/jake/unikraft/kraftkit/cmdfactory/builder.go:689 +0x19d
kraftkit.sh/cmdfactory.bind.func1(0xc000524900?, {0xc000112190, 0x1, 0x5})
	/home/jake/unikraft/kraftkit/cmdfactory/builder.go:689 +0x19d
kraftkit.sh/cmdfactory.bind.func1(0x0?, {0xc000112190, 0x1, 0x5})
	/home/jake/unikraft/kraftkit/cmdfactory/builder.go:689 +0x19d
kraftkit.sh/cmdfactory.bind.func1(0xc00053bd80?, {0xc000112190, 0x1, 0x5})
	/home/jake/unikraft/kraftkit/cmdfactory/builder.go:689 +0x19d
kraftkit.sh/cmdfactory.execute(0xc000524900, {0xc00034bef0, 0x5, 0x5})
	/home/jake/unikraft/kraftkit/cmdfactory/builder.go:231 +0x30b
kraftkit.sh/cmdfactory.executeC(0xc000004600)
	/home/jake/unikraft/kraftkit/cmdfactory/builder.go:291 +0x205
kraftkit.sh/cmdfactory.Main({0x2615568?, 0xc00049ef60}, 0xc000004600)
	/home/jake/unikraft/kraftkit/cmdfactory/builder.go:322 +0x73
main.main()
	/home/jake/unikraft/kraftkit/cmd/kraft/kraft.go:155 +0x549
```

--- a/machine/network/bridge/v1alpha1.go
+++ b/machine/network/bridge/v1alpha1.go
@@ -446,6 +446,10 @@ func (service *v1alpha1Network) Get(ctx context.Context, network *networkv1alpha
 		return network, err
 	}
 
+	if len(addrs) == 0 {
+		return network, fmt.Errorf("bridge %s has no ip address", network.Name)
+	}
+
 	network.Spec.Driver = "bridge"
 	network.Spec.Gateway = addrs[0].IP.String()
 	network.Spec.Netmask = net.IP(addrs[0].Mask).String()

